src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -82,6 +82,142 @@ private void VerifyTypeIL(CSharpCompilation compilation, string typeName, string
             CompileAndVerify(compilation).VerifyTypeIL(typeName, expected);
         }
 
+        [Fact] // PROTOTYPE(semi-auto-props): Test with initializer when they're supported.
+        public void TestInInterface()
+        {
+            var comp = CreateCompilation(@"
+public interface I
+{
+    public int P { get => field; }
```

Comment:
>field [](http://example.com/codeflow?start=26&length=5)

We should have a test with multiple references for the same `field`. Perhaps with one in the setter too, etc.